.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,6 +10,13 @@ Please answer these questions before submitting a bug report.
 
 ### What version of Node are you using?
 
+### What code are you using to set up your tracing?
+
+```typescript
+// Please provide the code used to set up your MeterProvider/TracerProvider
+// and register it for use with the API
```

Comment:
the copy especially in line 17 sounds a bit weird for me - at least this is how I understand that :/. Can this be simplified more, for example :
```typescript
  // Please copy your code example here
```

Reply:
I want to make it clear that we're asking for the code used to configure their tracers/meters, not the code that uses them.

Reply:
Since some people might use the `sdk` package which doesnt expose directly them (with a simple usage), i would prefer saying something along the lines of `Please provide the code you used to setup the OpenTelemetry SDK", WDYT ?